docs/aerodromes/classc/Nowra.md

```diff
@@ -4,7 +4,146 @@
+
+## Miscellaneous
+### Circuit Procedures
+Circuit operations at Nowra occur within the CIRA, a `5NM` radius of Nowra, from the surface to `2000ft`.
+
+- Fixed wing: `2000ft`
+- Rotary wing: `1200ft`
+- Flight Deck Procedural Trainer: not above `700ft`
+
+### Circuit Direction
+| Runway | Direction |
+| ------ | ----------|
+| 08     | Right  |
+| 21     | Right |
+| 03     | Left |
+| 26     | Left |
+
+## Helicopter Operations
+
+### Helicopter Spots
+
+The following helicopter spots are designated on various taxiways at Nowra aerodrome to facilitate visual helicopter arrivals and departures day and night.
+
+These spots are physically marked with bidirectional numbers in addition to the standard “H” marking:
+
+* Spot-1 (Taxiway C2)
+* Spot-2 (Taxiway C3)
+* Spot-3 (Taxiway B4)
+* Spot-4 (Taxiway B4)
+* Spot-5 (Taxiway B3)
+* Spot-6 (Taxiway B3)
+* Spot-7 (Taxiway A2)
+* Spot-8 (Taxiway A2)
+
+<figure markdown>
+![NW VFR CRD](img/nwrotaryslots.png){ width="700" }
+<figcaption>NW VFR CRD</figcaption>
+</figure>
+
+
+Helicopter departures for aircraft not requiring the use of a runway shall be conducted from the helicopter spots.  Helicopter spots shall be used by all departures and arrivals. Helicopters must remain at their designated slot until given a clearance from GND or TWR.
+
+Take-offs and landings to the Helicopter Spots should be parallel to the duty RWY direction. Normally the most upwind Helicopter Spot parallel to the RWY is designated for departures, and the downwind Helicopter Spot designated for arrivals. Helicopter Spots have been designed to allow simultaneous arrivals and departures of an adjacent Helicopter Spot.
+
+Post landing at a Helicopter Spot, helicopters shall not vacate the Spot until in receipt of a taxi clearance from ATC (SMC)
+
+### Duty runway Spots
+| Active Runway | Departure Spots (Order of Preference) | Arrival Spots (Order of Preference) |
+|---------------|----------------------------------------|-------------------------------------|
+| Runway 08     | Spot-2 (C3), Spot-1 (C2)               | Spot-8 (A2), Spot-7 (A2)            |
+| Runway 26     | Spot-8 (A2), Spot-7 (A2)               | Spot-2 (C3), Spot-1 (C2)            |
+| Runway 03     | Spot-6 (B3), Spot-5 (B3)               | Spot-4 (B4), Spot-3 (B4)            |
+| Runway 21     | Spot-4 (B4), Spot-3 (B4)               | Spot-6 (B3), Spot-5 (B3)            |
+
+
+## Coordination
+NWA TCU
+
+'Next' coordination is required from NWA ADC to NWA TCU for all aircraft.
+
+
+!!! example
+    <span class="hotline">**NWA ADC** -> **NWA TCU**</span>: "Next, ASY01, runway 08"  
+    <span class="hotline">**NWA TCU** -> **NWA ADC**</span>: "ASY01, Assigned Heading Right 030, unrestricted"  
+    <span class="hotline">**NWA ADC** -> **NWA TCU**</span>: "Assigned Heading Heading Right 030, ASY01" 
```

Comment:
This would be a left turn

Reply:
We'll also need to figure out what coord is required for a helicopter departure from a heli spot. I'm in favour of just treating it as a runway, for coord purposes, so you'd make a next call indicating the runway direction they're departing in (e.g. 'Next, XXX123, Runway 08'). An example in this situation might be helpful too

Reply:
Helicopters visual line up on their heli spot. Then depart with the direct of the duty runway is the way it's worded in the ersa and sop so helicopters would be the same i imagine.